src/main/java/com/spotinst/sdkjava/model/bl/admin/messageCenter/GetMessage.java

```diff
@@ -0,0 +1,375 @@
+package com.spotinst.sdkjava.model.bl.admin.messageCenter;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class GetMessage {
+    //region Members
+    @JsonIgnore
+    private Set<String> isSet;
+    private String title;
+    private String content;
+    private String stickyUntil;
+    private String type;
+    private String mainIcon;
+    private String hyperlink;
+    private String secondaryIcon;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+    private Date timestamp;
+    private String audienceSpecification;
```

Comment:
Confirm if these 3 fields are needed in the response object.

Reply:
Not used Removed